OpenRA.Mods.Common/Traits/Player/StrategicVictoryConditions.cs

```diff
@@ -109,8 +109,8 @@ void ITick.Tick(Actor self)
 
 		public void OnPlayerLost(Player player)
 		{
-			foreach (var a in player.World.Actors.Where(a => a.Owner == player))
-				a.Kill(a);
+			foreach (var a in player.World.ActorsWithTrait<INotifyOwnerLost>().Where(a => a.Actor.Owner == player))
```

Comment:
then you can just add `&& !a.Trait.IsDisabled` in these `.Where`s

Reply:
This nit ended up unaddressed.

Reply:
I added the IsTraitDisabled check to trait itself because i used a trait interface (in case someone wants to add a custom trait about owner losing).

Reply:
Hmm, okay, fine by me.